pom.xml

```diff
@@ -34,6 +34,10 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <nifi.version>1.9.0</nifi.version>
+        <aws-java-sdk.version>1.11.327</aws-java-sdk.version>
```

Comment:
Tim made a good point that a lot of the dependencies that were previously used aren't necessary anymore because we went the dynamic property route. So now the code's been updated to only use the java-sdk-cloudwatch dependency